perses/rjmc/geometry.py

```diff
@@ -238,16 +242,16 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
 
             if direction == 'forward':
                 pdbfile = open('%s-initial.pdb' % prefix, 'w')
-                PDBFile.writeFile(top_proposal.old_topology, old_positions, file=pdbfile)
+                PDBFile.writeFile(top_proposal.old_topology, old_sampler_state.positions, file=pdbfile)
```

Comment:
Since these writing methods will omit the box vectors,  which may be important in understanding what is going on in solvated systems, perhaps we want to add a method to `SamplerState` that allows us to obtain an MDTraj `Trajectory` object to write out to PDB? Or for now, band-aid this (and related functions) by providing the box vectors too?

Reply:
Good point. The issue with that is that the `SamplerState` lacks a `Topology` object (as does the `ThermodynamicState`. So I'd guess the method would look something like `SamplerState.to_trajectory(topology: simtk.openmm.app.Topology)`